bkrdoc/analysis/parser/conditions_for_commands.py

```diff
@@ -149,8 +149,13 @@ def is_rlgetdistro_command(self, command):
     def is_rlshowrunningkernel_command(self, command):
         return command[0:len("rlShowRunningKernel")] == "rlShowRunningKernel"
 
-    def is_phase(self, line):
-        return line[0:len("rlphasestart")] == "rlPhaseStart"
+    def is_phase_start(self, line):
+        return line == "rlPhaseStart"
+
+    def is_phase_startxxx(self, line):
+        return any([self.is_phase_clean(line),
+                    self.is_phase_setup(line),
+                    self.is_phase_test(line)])
 
     def is_phase_clean(self, line):
         return line[0:len("rlphasestartcleanup")] == "rlPhaseStartCleanup"
```

Comment:
i only changed phase_start because it did not comply with the manual, i did not touch the rest
i'll go ahead and do that now